browser/net/brave_site_hacks_network_delegate_helper_unittest.cc

```diff
@@ -267,4 +268,86 @@ TEST_F(BraveSiteHacksNetworkDelegateHelperTest,
   });
 }
 
+TEST_F(BraveSiteHacksNetworkDelegateHelperTest, QueryStringUntouched) {
+  std::vector<GURL> urls({
+    GURL("https://example.com/"),
```

Comment:
to save some bytes, I think you can omit `GURL` here just raw string literals should work. Same works for the next test

Reply:
Done.